app/controllers/api/v2/access_requests_controller.rb

```diff
@@ -1,19 +1,29 @@
 module API
   module V2
     class AccessRequestsController < API::V2::ApplicationController
-      before_action :build_access_request
+      before_action :build_access_request, except: :index
```

Comment:
Might be more explicit to say use `only` rather than `except` i.e. `before_action :build_access_request, only: :show`

Reply:
Great. Just changed it to 
` before_action :build_access_request, only: :approve`
Thanks Dan.